app/src/main/java/org/wikipedia/views/PlainPasteEditText.kt

```diff
@@ -48,7 +36,7 @@ open class PlainPasteEditText : TextInputEditText {
     }
 
     override fun onCreateInputConnection(outAttrs: EditorInfo): InputConnection? {
-        inputConnection = super.onCreateInputConnection(outAttrs)
+        val inputConnection = super.onCreateInputConnection(outAttrs)
```

Comment:
Would it be possible to move `super.onCreateInputConnection(outAttrs)` to line `48`?

Reply:
yep!